security/cmd/istio_ca/main.go

```diff
@@ -192,11 +228,6 @@ func initCLI() {
 	flags.BoolVar(&opts.selfSignedCA, "self-signed-ca", false,
 		"Indicates whether to use auto-generated self-signed CA certificate. "+
 			"When set to true, the '--signing-cert' and '--signing-key' options are ignored.")
-	flags.DurationVar(&opts.selfSignedCACertTTL, "self-signed-ca-cert-ttl", cmd.DefaultSelfSignedCACertTTL,
-		"The TTL of self-signed CA root certificate.")
-	flags.DurationVar(&opts.selfSignedRootCertCheckInterval, "self-signed-root-check-interval", cmd.DefaultSelfSignedRootCertCheckInterval,
-		"The interval that self-signed CA checks its root certificate expiration time and rotates root certificate. "+
-			"Should not be shorter than one minute.")
```

Comment:
If you change these, you will need to change the helm templates. Please only change what is needed in this PR :)

Reply:
Never mind. It seems these flags are not exposed in helm.